modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -64,7 +64,7 @@ spec:
     - ReadWriteMany
   resources:
     requests:
-  	storage: 10Gi
+    storage: 10Gi
```

Comment:
@duanwei33 @chao007  Could you check the change here ?

Reply:
@jboxman Hi, thanks for the finding, I guess previous line using the "tab" instead of "space", but the current fix is not good one, could you help update like this? 
  resources:
    requests:
      storage: 10Gi

@liangxia Thanks to involve us.

Reply:
"storage" is the next level for "requests".

Reply:
@duanwei33, updated, thanks!